tests/integration/04.Edge.Tests.ps1

```diff
@@ -89,6 +89,40 @@ Describe "Edge" {
         disconnect-nsxserver
     }
 
+    Context "Edge Status" {
+
+        BeforeAll{
+            if ( -not ( Get-NsxEdge $name ) ) {
+                New-NsxEdge -Name $name -Interface $vnics[0],$vnics[1],$vnics[2] -Cluster $cl -Datastore $ds -password $password -tenant $tenant -enablessh -Hostname "pestertest"
+            }
+        }
+
+        It "Get Edge Status" {
+            $status = Get-NsxEdge $name | Get-NsxEdgeStatus
+            $status | should not be $null
+            $status.systemStatus| should be "good"
```

Comment:
Rather than checking for a specific value, you should test for the existence of the property as the values could be different depending on the status of the edge in the test suite